src/main/java/de/doubleslash/keeptime/view/ReportController.java

```diff
@@ -151,6 +164,10 @@ private void updateReport(final LocalDate newvalue) {
 
       this.currentDayTimeLabel.setText(DateFormatter.secondsToHHMMSS(currentSeconds));
       this.currentDayWorkTimeLabel.setText(DateFormatter.secondsToHHMMSS(currentWorkSeconds));
+
+      controller = new Controller(model, dateProvider);
```

Comment:
we should not create new instances of the controller. if the controller is needed we should give a reference to it like for the model